content/chainguard/chainguard-images/faq.md

```diff
@@ -14,9 +14,9 @@ toc: true
 ---
 
 ### Are Chainguard Images based on Alpine?
-Some of our images are still based on Alpine, but going forward the majority will be based on  [Wolfi](/open-source/wolfi/), a Linux _undistro_ we built specifically to address software supply chain security issues.
+No. Chainguard images are based on [Wolfi](/open-source/wolfi/), a Linux _undistro_ we built specifically to address software supply chain security issues.
```

Comment:
What about:

Chainguard images are based on [Wolfi](/open-source/wolfi/), a Linux _undistro_ we built specifically to address software supply chain security issues. We do have some images with Alpine based variants in order to support musl or unusual architectures.